docling_core/types/doc/document.py

```diff
@@ -1309,6 +1310,17 @@ def export_to_dict(self) -> Dict:
         """export_to_dict."""
         return self.model_dump(mode="json", by_alias=True, exclude_none=True)
 
+    def save_to_json_file(self, document_json_path, indent: int=4) -> int:
+        """export_to_json."""
+        with open(document_json_path, 'w') as f:
+            return f.write(self.model_dump_json(indent=indent))
```

Comment:
Please don't use `model_dump_json` here, but something like `json.dumps(self.export_to_dict())`. The `export_to_dict` method already ensures that important pydantic serialization options are set.

Reply:
Done, thank you!